contracts/Comet.sol

```diff
@@ -201,6 +201,32 @@ contract Comet is CometMainInterface {
         (asset14_a, asset14_b) = getPackedAssetInternal(config.assetConfigs, 14);
     }
 
+    modifier nonReentrant() {
+        _nonReentrantBefore();
+        _;
+        _nonReentrantAfter();
```

Comment:
Let's inline these

Reply:
ok will remove the modifier function, and inline the functions into TransferIn and TransferOut.